examples/routing_service/shmem_udp_gateway/README.md

```diff
@@ -0,0 +1,172 @@
+# SHMEM <--> UDP Gateway
+
+This repository contains the necessary files to run the RTI Routing Service
+example from the "How does Connext help you meet the constraints of the
+hospital's IT department?" blogpost. Using RTI DDS Ping as a pub/sub
+application and Routing Service you will be able to create a gateway between
+a Shared Memory domain and a UDP domain that uses only 3 ports. There is also
+a second configuration option using RTI Cloud Discovery Service to use Unicast.
```

Comment:
Without fully duplicating the blogpost, can you add a short motivation for why you would want a gateway here?